Source/Scene/Scene.js

```diff
@@ -810,18 +845,23 @@ define([
          */
         this.maximumRenderTimeChange = defaultValue(options.maximumRenderTimeChange, 0.0);
         this._lastRenderTime = undefined;
+        this._frameRateMonitor = undefined;
 
         this._removeRequestListenerCallback = RequestScheduler.requestCompletedEvent.addEventListener(requestRenderAfterFrame(this));
         this._removeTaskProcessorListenerCallback = TaskProcessor.taskCompletedEvent.addEventListener(requestRenderAfterFrame(this));
         this._removeGlobeCallbacks = [];
 
+        var viewport = new BoundingRectangle(0, 0, context.drawingBufferWidth, context.drawingBufferHeight);
+
+        this._view = new View(this, this._camera, viewport);
```

Comment:
Can you have the view own the camera? You could have `scene.camera` get/set `scene._view.camera` until there are multiple views.